src/vm.cpp

```diff
@@ -32,9 +32,12 @@ int BinaryenVM::execute()
 {
   Module module;
   
+  //Copy over bytecode instead of type-punning to avoid breaking alias rules
+  vector<char> const bytecode(code.begin(), code.end());
```

Comment:
What alias rules exactly?

Reply:
The compiler does not allow me to reinterpret_cast vector<uint8_t> to vector<char> anymore.